tests/mmm/test_base.py

```diff
@@ -131,10 +132,32 @@ def build_model(self, data, **kwargs):
     return mmm
 
 
+class TestBaseMMM:
+    def test_bad_inheritance(self) -> None:
```

Comment:
Nitpick: this seems unnecessary. It's testing what the `abstractmethod` decorator is supposed to do. 
https://docs.python.org/3/library/abc.html#abc.abstractmethod